packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -63,13 +63,15 @@ export class IndexedDbMutationQueue implements MutationQueue {
    * Caches the document keys for pending mutation batches. If the mutation
    * has been removed from IndexedDb, the cached value may continue to
    * be used to retrieve the batch's document keys. To remove a cached value
-   * locally, `removeCachedMutationKeys()` should be invoked.
+   * locally, `removeCachedMutationKeys()` should be invoked either directly
+   * or through `removeMutationBatches()`.
    *
-   * During multi-tab, this cache is used by secondary clients to replay
-   * mutations after they were acknowledged or rejected by the primary client.
+   * After the primary client acknowledged or rejected a mutation during
```

Comment:
acknowledged => acknowledges, rejected => rejects

Perhaps instead of "during multi-tab" just preface the sentence with "With multi-tab, "

Reply:
Done